example/android/app/build.gradle

```diff
@@ -21,13 +22,9 @@ if (flutterVersionName == null) {
     flutterVersionName = '1.0'
 }
 
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
-
 android {
     compileSdkVersion flutter.compileSdkVersion
-    ndkVersion flutter.ndkVersion
+    ndkVersion = "26.1.10909125"
```

Comment:
Are we supposed to hardcode an ndk version?

I'm asking because I don't know, I'm not even really sure how the NDK is versioned :D

Reply:
I have not yet seen evidence that discourages hardcoding an ndk version. The example app gives me the following warning and a suggested fix:

```
Your project is configured with Android NDK 23.1.7779620, but the following plugin(s) depend on a different Android NDK version:
- integration_test requires Android NDK 26.1.10909125
- webcrypto requires Android NDK 26.1.10909125
Fix this issue by using the highest Android NDK version (they are backward compatible).
Add the following to C:\Users\Hamdaan\Desktop\downloads\webcrypto.dart\example\android\app\build.gradle:

    android {
        ndkVersion = "26.1.10909125"
        ...
    }
```

This is the [issue](https://github.com/flutter/flutter/issues/147806) where the message format was introduced.

The app should ideally use the NDK installed on the local machine using `flutter.ndkVersion`. I experimented a little and found out that `flutter run` will install the required NDK version if a lower version is found or if NDK is not installed:

```
Checking the license for package NDK (Side by side) 26.1.10909125 in ....   
License for package NDK (Side by side) 26.1.10909125 accepted.
Preparing "Install NDK (Side by side) 26.1.10909125 v.26.1.10909125".
```

Hardcoding the NDK version is simply a suggested fix, but flutter would install the minimum required NDK on the go even if I specify a higher NDK version, say - `27.0.12077973`
For some reason this did not happen the first time I made this edit.  Considering all this imo it would be best if we leave this to `flutter.ndkVersion`.